cvat/apps/documentation/user_guide.md

```diff
@@ -699,120 +699,101 @@ Go to the next/previous frame with a predefined step. Shortcuts:
 ![](static/documentation/images/image037.jpg)
 
 ---
-Go to the next/previous frame with step equals to 1. Shortcuts: ``D`` — previous, ``F`` — next.
 
-![](static/documentation/images/image040.jpg)
+### Top Panel
 
----
-Play the sequence of frames or the set of images. Shortcut: ``Space`` (change at ``Open Menu`` —> ``Settings`` —> ``Player Speed``).
-
-![](static/documentation/images/image041.jpg)
+![](static/documentation/images/image035.jpg)
 
 ---
-Go to specified frame. Press ``~`` to focus on the element.
-
-![](static/documentation/images/image060.jpg)
 
----
-**Open Menu** button
+#### Menu button
 
 It is the main menu of the annotation tool. It can be used to download, upload and remove annotations.
 
 ![](static/documentation/images/image051.jpg)
 
 Button assignment:
 
-- ``Open Task`` — open task in cvat dashboard
-- ``RunReID Merge`` — The ReID application uses deep learning model to perform
-  an automatic merging of bounding boxes between neighbor frames. You can use
-  "Merge" and "Split" functionality to edit automatically generated annotation
-  [read more](/cvat/apps/reid).
-- ``Dump Annotation`` — download annotations from the task
-- ``Upload Annotation`` — uploading annotations to the task
-- ``Remove Annotation`` — remove annotations from current task
-- ``Settings`` — open the settings menu
-- ``Fullscreen Player`` — fullscreen player mode
-- ``Switch AAM`` — switch to attribute annotation mode
-- ``Help`` — open the shortkeys
-- ``Save Work`` — save annotations for the current job. The button has an
-  indication of the saving process
-
-It also shows statistics about the current task, for example:
-- task name
-- status of the task: ``annotation``, ``validation`` or ``completed task``
-- technical information about task
-- number of created bounding boxes, sorted by labels (e.g. vehicle, person) and
-type of annotation (polygons, boxes, etc.)
+- ``Dump Annotations`` — downloads annotations from a task.
+- ``Upload Annotations`` — uploads annotations into a task.
+- ``Remove Annotations`` — removes annotations from the current job.
+- ``Open the task`` — opens a page with details about the task.
```

Comment:
The screenshot doesn't correspond the items. For example, `export as a dataset is missing`